app/src/main/java/com/evercheck/flagly/developeroptions/FeatureFlagHandlerPresenter.kt

```diff
@@ -17,6 +18,8 @@ class FeatureFlagHandlerPresenter @Inject constructor(
 ) : FeatureFlagActivityContract.Presenter, CoroutineScope {
 
     private val job: Job = SupervisorJob()
+    private var search: String = ""
```

Comment:
this global var is not necessary

Reply:
If necessary, since when updating the status, it consults all the information again and the filter would be lost